server/server.js

```diff
@@ -117,8 +122,9 @@ app.get('/api/fav/:user_email', async (req, res) => {
 //Get recommendation using openAI API
 const openai = new OpenAI({ apiKey: process.env.openai_key });
 const getChat = async function (req, res, next) {
+  const purpose = req.query.purpose; // Adjust this based on your client request structure
   const completion = await openai.chat.completions.create({
-    messages: [{ role: "system", content: "Please provide 5 recommendated movies as a JSON string that starts with [, and ends with ], representing an array of objects. Each recommendation object should have three properties: name, year, summary. The recommendation is for someone who is age 40 and likes action movies. " }],
+    messages: [{ role: "system", content: `Please provide 10 recommendated movies as a JSON string that starts with [, and ends with ], representing an array of objects. Each recommendation object should have three properties: name, year, summary. The recommendation is for someone whose purpose of watching movie is ${purpose}.`  }],
```

Comment:
```suggestion
    messages: [{ role: "system", content: `Please provide 10 recommended movies as a JSON string that starts with [, and ends with ], representing an array of objects. Each recommendation object should have three properties: name, year, summary. The recommendation is for someone whose purpose of watching movie is ${purpose}.`  }],
```

Reply:
Thank you for pointing it out. Just fix it.